HQSmokeTests/testPages/users/group_page.py

```diff
@@ -44,15 +44,18 @@ def add_group(self):
         assert self.is_visible_and_displayed(self.group_created_success), "Group not created successfully"
         print("Group Added")
 
-    def add_user_to_group(self):
-        self.send_keys(self.users_drop_down, "username_" + fetch_random_string())
+    def add_user_to_group(self, username):
+        self.send_keys(self.users_drop_down, username)
         self.wait_to_click(self.select_user)
         self.wait_to_click(self.update_button)
+        group_id_value = self.driver.current_url.split("/")[8]
+        print(group_id_value)
```

Comment:
@nsaxena03 instead of using [8], try using [-1]. The position of 8 might get changed in future, but if we are sure that the group id will be at the last position then use [-1].

Reply:
Hi @kbo001, I have made the changes as per your feedback. And pushed the code for review.